content/docs/_index.md

```diff
@@ -9,15 +9,18 @@ aliases:
 
 Bank-Vaults provides the following tools for Hashicorp Vault to make its use easier and more automated:
```

Comment:
```suggestion
We provide the following tools for Hashicorp Vault to make its usage easier and more automated:
```

Reply:
The main idea is understanding what `bank-vaults` represents across the docs: 
- as a project, it offers multiple tools
- as a repo, only the CLI tool